shared/message/input/MotionCapture.proto

```diff
@@ -99,16 +100,19 @@ message MotionCapture {
     /// Encoded timestamp - 0 := the first timecode. hour:minute:second:frame
     /// bytes, in binary coded decimal see
     /// https://en.wikipedia.org/wiki/SMPTE_timecode
-    uint32 timecode         = 3;
-    uint32 timecode_sub     = 4;
-    double natnet_timestamp = 5;
+    uint32 timecode                = 3;
+    uint32 timecode_sub            = 4;
+    double natnet_timestamp        = 5;
+    double mid_exposure_timestamp  = 6;
+    double data_recieved_timestamp = 7;
+    double transmit_timestamp      = 8;
```

Comment:
Be aware that when you reorder the numbers of protocol buffers like this you break any previous nbs files you have recorded. If you're doing it in the future for something you have recordings of, you can still put the fields here, but you need to use the bigger numbers.